controller/src/command.rs

```diff
@@ -346,7 +348,11 @@ pub fn receive(
 	g_args: &GlobalArgs,
 	args: ReceiveArgs,
 ) -> Result<(), Error> {
-	let adapter = FileWalletCommAdapter::new();
+	let adapter = match args.method.as_str() {
+		"file" => FileWalletCommAdapter::new(),
```

Comment:
Our plan is to deprecate "file" then, right? I can't think of any reason we would want file support when we have something as simple as a string.

Reply:
Not sure I understand, why would we want to deprecate the ability to save a slate as a file?

Reply:
This fully replaces every use case for file send/receive. One of our many usability issues right now is the fact that there are so many different ways to send/receive grin. Certain exchanges only support certain methods, and the same goes for wallets. It may sound counter-intuitive, but the way we make grin more usable is to support fewer transfer methods, not more.

Reply:
Still not with you here. Are you suggesting that anyone using the command line wallet should have to cut and paste strings into the command line instead of being allowed to specify a file name, and everyone creating a new transaction for email send (for instance) would either have to cut/paste or pipe output into a file?

Reply:
Sure, seems easy enough. Regardless, I'm not as worried about the 'file' method as having a separate serialization format for files. If we're going to serialize slates to base64 for files too, then I'm ok with keeping it.

Reply:
we can keep the file as is for now and have it output a hex or b64 file after the hard fork